src/Components/Facility/ConsultationForm.tsx

```diff
@@ -1180,29 +1179,25 @@ export const ConsultationForm = (props: any) => {
                 m<sup>2</sup>
               </div>
               {/* End of Telemedicine fields */}
-              <div className="mt-4 flex justify-between">
-                <Button
-                  color="default"
-                  variant="contained"
+              <div className="mt-4 sm:flex grid sm:justify-between">
+                <ButtonV2
+                  variant="secondary"
                   type="button"
+                  className="mb-2 sm:mb-0"
                   onClick={() =>
                     navigate(`/facility/${facilityId}/patient/${patientId}`)
                   }
                 >
-                  Cancel{" "}
-                </Button>
-                <Button
-                  color="primary"
-                  variant="contained"
+                  Cancel
+                </ButtonV2>
+                <ButtonV2
+                  variant="primary"
                   type="submit"
-                  style={{ marginLeft: "auto" }}
-                  startIcon={
-                    <CheckCircleOutlineIcon>save</CheckCircleOutlineIcon>
-                  }
                   onClick={(e) => handleSubmit(e)}
                 >
+                  <i className="fa-regular fa-circle-check" />
```

Comment:
@Ritesh-Aggarwal we are eventually moving out of font awesome and material icons.
(reference: https://github.com/coronasafe/care_fe/issues/3850)

We shall be using icons from unicons, or import svg if icon is not present in unicons library.

```suggestion
                  <i className="uil uil-check" />
```